CSharpBasics/Selenuim_Basics/EpamPagesWithWebElementsTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Selenuim_Basics
+{
+    [TestFixture]
+    public class EpamPagesWithWebElementsTests
+    {
+        private IWebDriver Driver { get; set; }
+
+        private const string _wordToSearchIsAutomation = "Automation";
+        private const string _wordToSearchIsBusinessAnalysis = "Business Analysis";
+        private const string _wordToSearchIsBlockchain = "Blockchain";
+
+        private const string _mainLinkPage = "https://www.epam.com/";
+        private const string _careersButtonXPath = "//*[@class ='top-navigation__item-link' and @href = '/careers']";
+        private const string _tabsWithCountriesXPath = "//*[contains(@class, 'tabs__title')]";
+        private const string _searchButtonMainPageXPath = "//*[@class = 'header-search__button header__icon']";
+        private const string _searchFieldXPath = "//input[@name = 'q']";
+        private const string _findButtonXPath = "//*[@class = 'header-search__submit']";
+        private const string _searchResultItemXPath = "//*[@class = 'search-results__item']";
+        private const string _titleOfFirstArticleOnSearchPageXPath = "//*[@class = 'search-results__item'][1]/h3/descendant::a";
+        private const string _titleOfArticleOnPageOpenedArticleXPath = "//h1[contains(@class, 'title-ui title--center')]";
+
+        [SetUp]
+        public void Initialazed()
+        {
+            Driver = new ChromeDriver();
+            Driver.Manage().Window.Maximize();
+            Driver.Navigate().GoToUrl(_mainLinkPage);
+        }
+
+        [Test]
+        public void CheckPageContainsThreeRegion()
+        {
+            var expectedResult = new List<string> { "AMERICAS", "EMEA", "APAC" };
+
```

Comment:
It is better to use only 1 empty string between blocks of code by code standards :)